alpha/declcfg/write_test.go

```diff
@@ -500,13 +500,13 @@ func removeJSONWhitespace(cfg *DeclarativeConfig) {
 	for ib := range cfg.Bundles {
 		for ip := range cfg.Bundles[ib].Properties {
 			var buf bytes.Buffer
-			json.Compact(&buf, cfg.Bundles[ib].Properties[ip].Value)
+			_ = json.Compact(&buf, cfg.Bundles[ib].Properties[ip].Value)
```

Comment:
maybe also `require.NoError` ?

Reply:
No test object available in this scope, or I would've. 🤷